src/ResourceManager/Profile/Commands.Profile/Subscription/GetAzureRMSubscription.cs

```diff
@@ -104,8 +104,24 @@ public override void ExecuteCmdlet()
             {
                 try
                 {
-                    var subscriptions = _client.ListSubscriptions(tenant);
-                    WriteObject(subscriptions.Select((s) => new PSAzureSubscription(s)), enumerateCollection: true);
+                    if (DefaultContext.Account.Type.Equals("ManagedService"))
```

Comment:
Please file an issue to add some automation for this - we have the infrastructure to mock managed service tests, and we should have some regression protection for thsi scenario.

Please make sure you have tested this in CloudShell.  LMK if you need some help in doign this.